### ISSUE TITLE
Create GET request for all events

### ISSUE THREAD

**Issue by ZakBrinlee** [ActoKids/AD440_W19_CloudPracticum#30]
API GET request for events.

Estimated time: 7 hours

Actual time:  8 hours

**Comment by ZakBrinlee** [comment]
Estimated Time: 7 hours

Time Spent: 8 hours as of 2/5/19

[Pull Request](https://github.com/ActoKids/api/pull/14)

[Wiki Page](https://github.com/ActoKids/api/wiki/GET-Request-for-Events)

GET all events:
`curl -v https://8cex032b53.execute-api.us-west-2.amazonaws.com/sprint2_test/v1/events`

GET event by id:
`curl -v https://8cex032b53.execute-api.us-west-2.amazonaws.com/sprint2_test/v1/events/1`

* The "id" is passed as the request path parameter.
* Event id 1 and 2 are valid and will return data. 
* Failures responses:  Error 400 for an invalid id, if the id field is blank a 502 error will return

**Comment by toddysm** [comment]
Few things:
- need a bit more as comments in the code
- What did you test?
- Who else tested this?

**Comment by ZakBrinlee** [comment]
I tested both Lambda Functions against my DynamoDB database with two current events. 

My code was reviewed and tested by Alex Clontz and Travis Tackett. Can review their comments and pull request with the following link: https://github.com/ActoKids/api/pull/14 

I reviewed the following pull-requests and tested their code. It worked and I did approve before the task was due. 

Alex Clontz: https://github.com/ActoKids/api/pull/9
Travis Tackett: https://github.com/ActoKids/api/pull/13
